.changes/next-release/Bug Fix-d47847cc-bbb7-4c26-a102-82943152654a.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer: patch java test file name MyClassTests.java will not be captured if file is not living in a test folder"
```

Do customers know what "patch java test file" means?

```suggestion
	"description": "CodeWhisperer: java test file name MyClassTests.java will not be captured if file is not living in a test folder"
```

---

sgg